test/jdk/javax/swing/JTable/KeyBoardNavigation.java

```diff
@@ -224,13 +173,128 @@ public void setValueAt(Object aValue, int row, int column) {
+                """;
+
+        final String LINUX_SPECIFIC = """
+                Tab, Shift-Tab - Navigate In.
+                Return/Shift-Return - Move focus one cell down/up.
+                Tab/Shift-Tab -  Move focus one cell right/left.
+                Up/Down Arrow - Deselect current selection;
+                                move focus one cell up/down
+                Left/Right Arrow - Deselect current selection;
+                                   move focus one cell left/right
+                PageUp/PageDown - Deselect current selection;
+                                  scroll up/down one  JViewport view;
+                                  first visible cell in current column gets focus
+                Home/End - Deselect current selection; move focus and view to
+                                     first/last cell in current row
+                F2 - Allows editing in a cell containing information without
+                     overwriting the information
+                Esc -  Resets the cell content back to the state it was in
+                       before editing started
+                Ctrl+A, Ctrl+/ - Select All
+                Ctrl+\\ - Deselect all
+                Shift-Up/Down Arrow -  Extend selection up/down one row
+                Shift-Left/Right Arrow - Extend selection left/right one column
+                Control-Shift Up/Down Arrow -  Extend selection to top/bottom of
+                                               column
+                Shift-Home/End -  Extend selection to left/right end of row
+                Shift-PageUp/PageDown - Extend selection up/down one view and
+                                        scroll  table
+                """;
+
+        final String MAC_SPECIFIC = """
+                Tab, Shift-Tab - Navigate In.
+                Return/Shift-Return - Move focus one cell down/up.
+                Tab/Shift-Tab -  Move focus one cell right/left.
+                Up/Down Arrow - Deselect current selection; move focus one cell
+                                up/down
+                Left/Right Arrow - Deselect current selection;
+                                   move focus one cell left/right
+                FN + Up Arrow/FN + Down Arrow - Deselect current selection;
+                                   scroll up/down one JViewport view;
+                                   first visible cell in current column gets focus
+                Control-FN+Up Arrow/FN+Down Arrow - Deselect current selection;
+                                                    move focus and view to
+                                                    first/last cell in current row
+                F2 - Allows editing in a cell containing information without
+                     overwriting the information
+                Esc -  Resets the cell content back to the state it was in
+                       before editing started
+                Ctrl+A, Ctrl+/ - Select All
```

Comment:
Suggestion:
At L273 you have given space before and after + but not here. I feel it should be uniform across the instruction.

Reply:
Updated.